include/z64skin.h

```diff
@@ -0,0 +1,85 @@
+#ifndef Z64_SKIN_H
+#define Z64_SKIN_H
+
+#include "z64animation.h"
+
+/**
+ * Holds a compact version of a vertex used in the Skin system
+ * It is used to initialise the Vtx used by an animated limb
+ */
+typedef struct {
+    /* 0x00 */ u16 index;
+    /* 0x02 */ s16 u; // u and v are texture coordinates
+    /* 0x04 */ s16 v;
```

Comment:
they are called s and t in the gbi, I guess they should be here too since they get directly assigned to `Vtx_tn#tc`

Reply:
i had thought about this too, but i figured since its a custom struct and not an actual vtx it would be fine to make things more "colloquial".
Can also see the issue with the inconsistency.

`s` and `t` with comments about them being u and v would probably be the best compromise